src/main/java/it/gov/pagopa/pu/bff/connector/auth/client/AuthzClient.java

```diff
@@ -0,0 +1,31 @@
+package it.gov.pagopa.pu.bff.connector.auth.client;
+
+import it.gov.pagopa.pu.auth.dto.generated.UserInfo;
+import it.gov.pagopa.pu.bff.connector.auth.config.AuthApisHolder;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.HttpClientErrorException;
+
+@Service
+@Slf4j
+public class AuthzClient {
+
+  private final AuthApisHolder authApisHolder;
+
+  public AuthzClient(AuthApisHolder authApisHolder) {
+    this.authApisHolder = authApisHolder;
+  }
+
+  public UserInfo getUserInfoFromMappedExternaUserId(String mappedExternalUserId, String accessToken) {
+    try{
+      return authApisHolder.getAuthzApi(accessToken)
+        .getUserInfoFromMappedExternaUserId(mappedExternalUserId);
+    } catch (HttpClientErrorException e) {
```

Comment:
it could return 404 (it should not happen, but if a user is removed it could...)
let's handle it returni null here and handle it in the mapper showing the mappedExternalUserdId instead of name+lastname concat